### ISSUE TITLE
Comments are removed from exported cells

### ISSUE THREAD

**Issue by sgugger** [fastai/nbdev#167]
@pete88b I was reading the diff wrong the other day. It looks like the recent changes with the magic command broke the following thing: when a cell as an export tag then a comment, the comment is removed. Example:
```
#export
# TODO: write this function
def func(x): pass
```
is exported in just
```
def func(x): pass
```

**Comment by pete88b** [comment]
Got you. I'll change this ASAP

**Comment by sgugger** [comment]
Fixed by #168
